### PR TITLE
Clarifications on downgrade and integrations

### PR DESCRIPTION
**Pull request by chouetz**
* Reword the text of the `datadog_agent_allow_downgrade` variable to prevent confusion. Enriched the notes about downgrade on other places of the document
* Brought clarification about integrations not to confuse the role of `datadog_check` and `datadog_integration`, and changed the paragraph name that was dupplicated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -327,20 +329,22 @@ To override the default behavior, set this variable to something other than an e
 
 To upgrade from Agent v6 to v7, use `datadog_agent_major_version: 7` to install the latest version or set `datadog_agent_version` to a specific version of Agent v7. Use similar logic to upgrade from Agent v5 to v6.
 
-#### Integrations
+#### Integrations installation
```
Comment (urseberry) [inline README.md]:
```suggestion
#### Integration installation
```

File: README.md [hunk 2]
```diff
@@ -327,20 +329,22 @@ To override the default behavior, set this variable to something other than an e
...
 
 **Available for Agent v6.8+**
 
-Use the `datadog_integration` resource to install a specific version of a Datadog integration. Keep in mind, the Agent comes with all the integrations already installed. This command is useful for upgrading a specific integration without upgrading the whole Agent. For more details, see [Integration Management][4].
+Use the `datadog_integration` resource to install a specific version of a Datadog integration. Keep in mind, the Agent comes with all the integrations ([core integrations][19]) already installed. This command is useful for upgrading a specific integration without upgrading the whole Agent. For more details, see [Integration Management][4].
```
Comment (urseberry) [inline README.md]:
```suggestion
Use the `datadog_integration` resource to install a specific version of a Datadog integration. Keep in mind, the Agent comes with the [core integrations][19] already installed. This command is useful for upgrading a specific integration without upgrading the whole Agent. For more details, see [integration management][4].
```

File: README.md [hunk 3]
```diff
@@ -327,20 +329,22 @@ To override the default behavior, set this variable to something other than an e
...
+
+If you want to configure an integration, please refer to the `datadog_checks` [paragraph][21]
```
Comment (urseberry) [inline README.md]:
```suggestion
If you want to configure an integration, refer to the `datadog_checks` [paragraph][21]
```

File: README.md [hunk 4]
```diff
@@ -77,7 +77,9 @@ To deploy the Datadog Agent on hosts, add the Datadog role and your API key to y
 
 ### Integrations
 
-To configure a Datadog integration (check), add an entry to the `datadog_checks` section. The first level key is the name of the check, and the value is the YAML payload to write the configuration file. Examples are provided below.
+To configure a Datadog integration (check), add an entry to the `datadog_checks` section. The first level key is the name of the check, and the value is the YAML payload to write the configuration file. Examples are provided below. 
+
+To install or remove an integration, please refer to the `datadog_integrations` [paragraph][22]
```
Comment (urseberry) [inline README.md]:
```suggestion
To install or remove an integration, refer to the `datadog_integrations` [paragraph][22]
```

File: README.md [hunk 5]
```diff
@@ -327,20 +329,22 @@ To override the default behavior, set this variable to something other than an e
...
 Available actions:
 
 - `install`: Installs a specific version of the integration.
 - `remove`: Removes an integration.
 
-##### Datadog Marketplace
+##### Third party integrations
 
-[Datadog Marketplace][15] integrations can be installed with the `datadog_integration` resource. **Note**: Marketplace integrations are considered to be "third party" and thus have to have `third_party: true` set - see the example below.
+[Datadog Community][20] and [Datadog Marketplace][15] integrations can be installed with the `datadog_integration` resource. **Note**: These integrations are considered to be "third party" and thus need `third_party: true` to be set - see the example below.
```
Comment (urseberry) [inline README.md]:
```suggestion
[Datadog community][20] and [Datadog Marketplace][15] integrations can be installed with the `datadog_integration` resource. **Note**: These integrations are considered to be "third party" and thus need `third_party: true` to be set—see the example below.
```

File: README.md [hunk 6]
```diff
@@ -385,7 +389,7 @@ To downgrade to a prior version of the Agent:
 
 **Notes:**
 
-- Downgrades are not supported for Windows platforms.
+- Downgrades are not supported for Windows platforms. On CentOS it requires Ansible 2.4+
```
Comment (urseberry) [inline README.md]:
```suggestion
- Downgrades are not supported for Windows platforms. On CentOS, downgrades require Ansible 2.4+
```

File: README.md [hunk 7]
```diff
@@ -63,7 +63,7 @@ To deploy the Datadog Agent on hosts, add the Datadog role and your API key to y
...
-| `datadog_agent_allow_downgrade`             | Set to `yes` to allow Agent downgrades on apt-based platforms (use with caution, see `defaults/main.yml` for details). **Note**: On Centos this only works with Ansible 2.4+.                                                                                                                                                                                                                                                                     |
+| `datadog_agent_allow_downgrade`             | Set to `yes` to allow Agent downgrade (use with caution, see `defaults/main.yml` for details). **Note**: Downgrades are not supported on Windows platforms. On Centos it only works with Ansible 2.4+.|
```
Comment (urseberry) [inline README.md]:
```suggestion
| `datadog_agent_allow_downgrade`             | Set to `yes` to allow Agent downgrade (use with caution, see `defaults/main.yml` for details). **Note**: Downgrades are not supported on Windows platforms. On CentOS, Agent downgrade only works with Ansible 2.4+.|
```

**Review by urseberry — COMMENTED** [review]
Thank you!

File: README.md (hunk 7 above)
Comment (KSerrania) [reply README.md]: The role only supports Ansible 2.6+, so I'd remove the lines about Ansible 2.4+.

**Review by bkabrda — APPROVED** [review]

**Pull request merged**